packages/publish-views/shapes.ttl

```diff
@@ -0,0 +1,98 @@
+PREFIX dcat: <http://www.w3.org/ns/dcat#>
+PREFIX void: <http://rdfs.org/ns/void#>
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX schema: <http://schema.org/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+prefix view: <https://cube.link/view/>
+
+<ViewShape>
+  a sh:NodeShape ;
+  sh:targetClass view:View, schema:Dataset ;
+  sh:property
+    [
+      sh:path rdf:type ;
+      sh:hasValue dcat:Dataset, void:Dataset ;
+    ],
+    [
+      sh:path schema:name ;
+      sh:minCount 1 ;
+      sh:nodeKind sh:Literal ;
+    ],
+    [
+      sh:path schema:identifier ;
+      sh:minCount 1 ;
+      sh:maxCount 1 ;
+      sh:nodeKind sh:Literal ;
+    ],
+    [
+      sh:path dcterms:issued ;
+      sh:maxCount 1 ;
+      sh:or
+        (
+          [ sh:datatype xsd:date ]
+          [ sh:datatype xsd:dateTime ]
+        ) ;
+      sh:severity sh:Warning ;
+    ],
+    [
+      sh:path dcterms:publisher ;
+      sh:nodeKind sh:IRI ;
+      sh:severity sh:Warning ;
+    ],
+    [
+      sh:path schema:alternateName ;
+      sh:nodeKind sh:Literal ;
+      sh:severity sh:Warning ;
+    ],
+    [
+      sh:path schema:description ;
+      sh:nodeKind sh:Literal ;
```

Comment:
min 0, max 1 . 
If it's the case

Reply:
In fact, `maxCount` may not be a good fit here in case we want multilingual annotations. I don't know, hence I kept this mostly unspecified